components/tabbed-nav/src/index.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+
+// styled components
+const NavNavContainerli = styled('div')`
+  display: block;
+  padding: 0;
+  margin: 0;
+`;
+
+const NavUl = styled('ul')`
+    list-style: none;
+    margin: 0;
+    border-top: 0;
+    padding: 0;
+    border-bottom: 2px solid #bfc1c3; 
+`;
+
+const Navli = styled('li')`
+  float: left;
+  margin: 0 3px 0 0;
+`;
+
+const NavLink = styled('a')`
+    font-family: "nta", Arial, sans-serif;
+    font-weight: 400;
+    text-transform: none;
+    cursor: pointer;
+    display: block;
+    border-bottom: none;
+    float: left;
+    height: 3.25em;
+    line-height: 3.25em;
+    margin-bottom: -2px;
+    padding: 0 40px;
+    border-top: ${props => props.active ? '2px solid #bfc1c3' : 'none'};
```

Comment:
There are more elegant ways of doing this - if you reference some of the other components like `Button` you'll see it de-structures props.

Also, we will need to swap these colour vars for govuk-colours, which is reference somewhere (check `Button` again for that too).